tachyon/zk/plonk/keys/proving_key.h

```diff
@@ -131,7 +131,7 @@ class ProvingKey : public Key<PCS> {
     // | 5 | 0         |
     // | 6 | 0         |
     // | 7 | 0         |
-    size_t usable_rows = prover->GetUsableRows();
+    RowIndex usable_rows = prover->GetUsableRows();
```

Comment:
I'm really not sure if we need to import row_index.h in this case.

Plus, we might have to decide header import rules since this kind of refac with new header makes it really hard to review.

Reply:
I agree. To check whether header include is correct manually is not productive. Some omitting header is maybe intended by owner. I think we need to comment where if header is not included where it should be.